3rdparty/python/requirements.txt

```diff
@@ -8,7 +8,7 @@ mox==0.5.3
 
 # Until https://github.com/pantsbuild/pex/issues/28 is addressed, we cannot
 # do pex<1.1.0.
-pex>=1.0.2,<1.0.999999
+pex>=1.0.3,<1.0.999999
```

Comment:
You damn github commenter!  Yeah - I see no reason not to pin.  Its highly unlikely a plugin writer outside the pants repo:
a. exists
b if they exist, that they add python tasks that use newer pex features than pants which now releases weekly.